tests/tensorflow/quantization/test_strip.py

```diff
@@ -52,3 +56,30 @@ def test_do_copy(do_copy):
         assert id(inference_model) != id(compression_model)
     else:
         assert id(inference_model) == id(compression_model)
+
+
+@pytest.mark.parametrize("strip_model", (True, False, None))
+def test_nncf_quantize_strip(strip_model):
+    model = get_basic_two_conv_test_model()
+
+    def transform_fn(data_item):
+        return data_item[0]
+
+    dataset = nncf.Dataset(get_const_target_mock_regression_dataset(img_size=4), transform_fn)
+
+    if strip_model is not None:
+        advanced_parameters = AdvancedQuantizationParameters()
```

Comment:
Can I do it this way? 
```py
advanced_parameters = AdvancedQuantizationParameters(strip_model=True)
```

Reply:
Yes, it`s equivalently. Update in tests.